### PR TITLE
Fix load action_controller/responder, test multiple rails versions

### LINKED ISSUE #19: getting error after install gem with rails 4.2
```
Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet/rails/action_pack_renderers.rb:3:in `require': cannot load such file -- action_controller/responder (LoadError)
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet/rails/action_pack_renderers.rb:3:in `<top (required)>'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet/railtie.rb:3:in `require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet/railtie.rb:3:in `<top (required)>'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet.rb:4:in `require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet.rb:4:in `<top (required)>'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:76:in `require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:76:in `block (2 levels) in require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:72:in `each'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:72:in `block in require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:61:in `each'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:61:in `require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler.rb:134:in `require'
	from /Users/egorgorodov/dev/ruby/goparcel/config/application.rb:7:in `<top (required)>'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.1.4/lib/rails/commands/commands_tasks.rb:146:in `require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.1.4/lib/rails/commands/commands_tasks.rb:146:in `require_application_and_environment!'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.1.4/lib/rails/commands/commands_tasks.rb:68:in `console'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.1.4/lib/rails/commands/commands_tasks.rb:40:in `run_command!'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.1.4/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'
```

Can we fix it?

p.s How i can generate xlsx and put result to file for storing locally?

### PR DESCRIPTION
**Pull request by Paxa**
Fixes #19

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,4 +4,10 @@ rvm:
   - 2.2.2
   - 1.9.3
   - jruby
+gemfile:
+  - gemfiles/Gemfile.rails-3.2
```
Comment (glebm) [inline .travis.yml]: These should live in `spec/gemfiles` which would also require `bundler_args` to work with caching. See for example https://github.com/jayroh/thredded/blob/master/.travis.yml

File: spec/rails_integration_spec.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require "to_spreadsheet/railtie"
```
Comment (glebm) [inline spec/rails_integration_spec.rb]: Please use single quotes for non-interpolated strings.

File: spec/rails_integration_spec.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+describe ToSpreadsheet::Railtie do
+
+  it "should register renderer" do
```
Comment (glebm) [inline spec/rails_integration_spec.rb]: This should says `'registers a renderer'`. "Should" is redundant.

**Comment by glebm** [comment]
Thanks a lot, looks good! Please address the comments and I will merge this in.

File: .travis.yml (hunk 1 above)
Comment (Paxa) [inline .travis.yml]: First I put Gemfiles in spec folder, but I didn't think about bundler_args so I didn't work, thanks, nice to know.

Do you mind to fix this and other notes about code style by yourself and merge it? I think you know better how it should looks like

Comment (glebm) [inline .travis.yml]: I can fix this myself but I don't have access to my dev machine until tomorrow.

**Comment by Paxa** [comment]
@glebm Ready

**Comment by glebm** [comment]
Awesome, thanks! New version coming soon.

**Pull request merged**